elife/aws-cli.sls

```diff
@@ -0,0 +1,6 @@
+aws-cli:
+    cmd.run:
```

Comment:
`cmd.run` should be a last resort. 
please switch to: https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pip_state.html#salt.states.pip_state.installed